libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/NativeLoginManager.kt

```diff
@@ -266,6 +271,146 @@ internal class NativeLoginManager(
         return requestBodyString.toRequestBody(mediaType)
     }
 
+    // region Salesforce Identity API Headless Registration Flow
+
+    override suspend fun startRegistration(
```

Comment:
Everything here in start registration should seem very similar to all the previous native authentication methods.